packaging/targets/trusty/etc/init/openhim-console.conf

```diff
@@ -0,0 +1,22 @@
+# OpenHIM server upstart config
+
+description "OpenHIM Console"
+
+#logs to /var/log/upstart/openhim-console.log
+console log
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+
+setuid openhim
+setgid openhim
+
+env PATH=$PATH:/home/openhim/.nvm/current/bin
+
+script
+  chdir /home/openhim/openhim-console
+  exec /home/openhim/openhim-console/node_modules/grunt-cli/bin/grunt serve
```

Comment:
We need to change this to not use grunt but rather copy the console files into Apache or another more production ready http-server. Copying into Apache seems easiest but how do we ensure that it doesn't overwrite anything? Should be expose it in an /openhim folder?